source/docs/software/advanced-controls/state-space/state-space-observers.rst

```diff
@@ -119,3 +119,8 @@ WPILib's Kalman Filter classes' constructors take a linear system, a vector of p
          :lines: 47-52
          :linenos:
          :lineno-start: 47
+
+Footnotes
+---------
+
+.. [1] In a real robot, noise comes from all sorts of sources. Stray electromagnetic radiation adds extra voltages to sensor readings, vibrations and temperature variations throw off inertial measurement units, gear lash causes encoders to have inaccuracies when directions change... all sorts of things. It's important to realize that, by themselves, each of these sources of "noise" aren't guaranteed to follow any pattern. Some of them might be the "white noise" random vibrations you've probably heard on the radio. Others might be "pops" or single-loop errors. Others might be nominally zero, but strongly correlated with events on the robot. However, the Central Limit Theorem shows mathematically that regardless of how the individual sources of noise are distributed, as we add more and more of them up their combined effect eventually is distributed like a Gaussian. Since we do not know the exact individual sources of noise, the best choice of a model we can make is indeed that Gaussian function.
```

Comment:
Can there be a link to a definition of Central Limit Theorem?

Reply:
![image](https://user-images.githubusercontent.com/4583662/188542463-ec063ace-3bc0-4774-940c-47ab8575ce06.png)
Done